share/spice/untappd/footer.handlebars

```diff
@@ -0,0 +1,7 @@
+<div class="untappd__footer tile__update__info two-line">
+    <span>{{abv}}% ABV</span>
+    <i class="sep"></i>
+    <span>{{ibu}} IBU</span>
+    <i class="sep"></i>
+    <span><i class="count__i"></i> {{ratings}}</span>
```

Comment:
I have no information actually. I think that there will no problem with it if there are good usage statistics. And it's hard to predict stats now.